textstat/textstat.py

```diff
@@ -305,9 +305,12 @@ def linsear_write_formula(self, text):
 
         text = ' '.join(text_list)
 
-        number = float(
-            (easy_word * 1 + difficult_word * 3)
-            / self.sentence_count(text))
+        try:
+            number = float(
+                (easy_word * 1 + difficult_word * 3) / self.sentence_count(text)
+            )
+        except ZeroDivisionError:
+            number = 0.0
```

Comment:
I think it might be best if we just `return 0.0` here. Continuing creates the inconstancy that `linsear_write_formula` will return `-1.0` if supplied with an empty string.

Reply:
Done